packages/qtwebengine.rb

```diff
@@ -1,29 +1,26 @@
 # Adapted from Arch Linux qt5-webengine PKGBUILD at:
 # https://github.com/archlinux/svntogit-packages/raw/packages/qt5-webengine/trunk/PKGBUILD
+# & LFS build documentation at https://www.linuxfromscratch.org/blfs/view/svn/x/qtwebengine.html
 
 require 'package'
 require_relative 'qtbase'
 
 class Qtwebengine < Package
   description 'Provides support for web applications using the Chromium browser project'
   homepage 'https://www.qt.io'
-  version '5.15.13'
+  version '5.15.14'
   @qt_pkgver = Qtwebengine.version.to_s.gsub(/-.*/, '')
   @qt_basever = Qtbase.version.to_s.gsub(/-.*/, '')
   license 'LGPL3 LGPL2.1 BSD'
-  compatibility 'x86_64 aarch64 armv7l'
-  source_url 'https://invent.kde.org/qt/qt/qtwebengine.git'
-  git_hashtag "v#{version}-lts"
+  compatibility 'x86_64'
+  source_url 'https://anduin.linuxfromscratch.org/BLFS/qtwebengine/qtwebengine-5.15.14.tar.xz'
+  source_sha256 '1475c004860585b22af13a2b719ef10992fad58f1c145bf5cb43fbc86a3a67d2'
```

Comment:
What is wrong with the git repo?

Reply:
As per the LFS docs, qtwebengine 5.15 is officially no longer supported for free as of next week, and needs a bunch of patched sources , so this puts it there.  See https://www.linuxfromscratch.org/blfs/view/svn/x/qtwebengine.html